lib/js/importers/AllInOneImporter.ts

```diff
@@ -0,0 +1,189 @@
+
+				if (hexToCssVariable[obj.values.hex] === undefined) {
+					hexToCssVariable[obj.values.hex] = colorPrefix + colorName;
+				}
+
+				result.push(colorPrefix + colorName + ': ' + obj.values.hex + ';');
+				result.push(colorPrefix + colorName + '-rgb' + ': ' + hexToRgb(obj.values.hex) + ';');
+
+				/** JSON object structure */
+				colorName = colorName.replace(/--/i, '').replace(/raw-/i, '');
+				const colorNameSplitted = colorName.split('-');
+				const category = (colorNameSplitted[1] === undefined) ? 'default' : colorNameSplitted[0];
+				const resultJsonObject: IResultJsonObject = {
+					id: binValues.destination + '_' + colorName,
+					label: colorName,
+					value: obj.values.hex,
+					weight: colorNameSplitted[2] ? parseInt(colorNameSplitted[2]) : parseInt(colorNameSplitted[1])
+				};
+
+				if (resultJson[category] === undefined) {
+					resultJson[category] = [];
+				}
+				resultJson[category].push(resultJsonObject);
+				resultJson[category].sort((a,b) => (parseInt(a.weight) > parseInt(b.weight)) ? 1 : ((parseInt(b.weight) > parseInt(a.weight)) ? -1 : 0))
+			}
+		});
+		saveColorFile(tokensFilesConfig.destinationPath + binValues.destination, result);
+		saveColorFileJSON(tokensFilesConfig.destinationPath + binValues.destinationJson, resultJson);
+
+	} catch (err) {
+		console.error(err)
+	}
+
+	return hexToCssVariable;
+};
+
+const ImportSingleTokenFile = (
+	binValues: configFileObject,
+	jsonSource: { data },
+	hexToCssVariable: Object,
+) => {
+	try {
+		let result: Array<string> = [];
+		let resultJson = {};
+
+		jsonSource.data.colors.forEach(obj => {
+			const patternDeprecated = /deprecated|Pattern|RAW|theme/i;
+			if (obj.name.match(patternDeprecated) === null) {
+				let tokenName = obj.name;
+				tokenName = tokenName.replace(/\//i, '-');
+
+				obj.values.hex = obj.values.hex.replace(/FFFFFF/gi, function(){
+					return 'fff';
+				});
+
+				if (obj.values.alpha !== 1) {
+					result.push('$' + tokenName + ': ' + 'rgba(var(' + hexToCssVariable[obj.values.hex] +'), ' + obj.values.alpha + ');');
+				} else {
+					result.push('$' + tokenName + ': ' + hexToCssVariable[obj.values.hex] + ';');
+				}
+
+
+
+				/** JSON object structure */
+				const tokenNameSplitted = tokenName.split('-');
+				const category = tokenNameSplitted[1];
+				const resultJsonObject: ITokenJsonObject = {
+					id: binValues.destination + '_' + tokenName,
+					label: tokenName,
+					value: hexToCssVariable[obj.values.hex],
+				};
+				if (resultJson[category] === undefined) {
+					resultJson[category] = [];
+				}
+				resultJson[category].push(resultJsonObject);
+			}
+		});
+		saveTokenFile(tokensFilesConfig.destinationPath + binValues.destination, result);
+		saveTokenFileJSON(tokensFilesConfig.destinationPath + binValues.destinationJson, resultJson);
+	} catch (err) {
+		console.error(err)
+	}
+}
+
+const saveTokenFileJSON  = (filepath: string, content: any) => {
```

Comment:
why `any`?

Reply:
I changed it, for one type of files is `content: Array<string>` for second one `content: Object`